### ISSUE TITLE
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name - multibutton accessibility

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#683]
**Describe the bug**
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name 
 - This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.

To Reproduce:
run: 
 -  `pa11y --debug --reporter json --standard WCAG2AA <url of component>`

example:
 -  `pa11y --debug --reporter json --standard WCAG2AA --ignore WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2  http://localhost:9002/iframe.html?id=coremultibutton--icononlyhorizontalmultipleselection'

**Components samples with that error**

http://localhost:9002/iframe.html?id=coremultibutton--icononlyhorizontalmultipleselection
`pa11y --runner htmlcs --runner axe --debug --reporter json --standard WCAG2AA --root-element "#root > div.Component-content-4" --ignore WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name http://localhost:9002/iframe.html?id=coremultibutton--icononlyhorizontalmultipleselection`

```
  "code": "button-name",
  "type": "error",
  "typeCode": 1,
  "message": "Buttons must have discernible text (https://dequeuniversity.com/rules/axe/3.3/button-name?application=axeAPI)",
  "context": "<button class=\"MuiButtonBase-root-44 MuiButton-root-18 HvButton-root-9 MuiButton-outlined-26 MuiButton-outlinedPrimary-27 HvButton-secondary-12 MultiButton-btnBase-150 MultiButton-btnSecondary-152 MultiButton-iconWidth-148 MultiButton-isSelected-153\"...",
  "selector": "#map",
  "runner": "axe",
  "runnerExtras": {
    "description": "Ensures buttons have discernible text",
    "impact": "critical",
    "help": "Buttons must have discernible text",
    "helpUrl": "https://dequeuniversity.com/rules/axe/3.3/button-name?application=axeAPI"
  }
```

http://localhost:9002/iframe.html?id=coremultibutton--icononlyhorizontalmultipleselection
```
  "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name?",
  "type": "error",
  "typeCode": 1,
  "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
  "context": "<button class=\"MuiButtonBase-root-44 MuiButton-root-18 HvButton-root-9 MuiButton-outlined-26 MuiButton-outlinedPrimary-27 HvButton-secondary-12 MultiButton-btnBase-150 MultiButton-btnSecondary-152 MultiButton-iconWidth-148 MultiButton-isSelected-153\"...",
  "selector": "#map",
  "runner": "htmlcs",

  "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
  "type": "error",
  "typeCode": 1,
  "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
  "context": "<button class=\"MuiButtonBase-root-44 MuiButton-root-18 HvButton-root-9 MuiButton-text-20 MuiButton-textPrimary-21 HvButton-ghost-14 MuiButton-flat-23 MuiButton-flatPrimary-24 MultiButton-btnBase-150 MultiButton-btnSecondary-152 MultiButton-iconWidth-...",
  "selector": "#location",
  "runner": "htmlcs",
```

http://localhost:9002/iframe.html?id=coremultibutton--icononlyhorizontalsingleselection
```
 "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
 "type": "error",
 "typeCode": 1,
 "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
 "context": "<button class=MuiButtonBase - root - 44 MuiButton - root - 18 HvButton - root - 9 MuiButton - outlined - 26 MuiButton - outlinedPrimary - 27 HvButton - secondary - 12 MultiButton - btnBase - 150 MultiButton - btnSecondary - 152 MultiButton - iconWidth - 148 MultiButton - isSelected - 153\\ "...",
 "selector": "#map",
 "runner": "htmlcs",

 "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
 "type": "error",
 "typeCode": 1,
 "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
 "context": "<button class= MuiButtonBase - root - 44 MuiButton - root - 18 HvButton - root - 9 MuiButton - text - 20 MuiButton - textPrimary - 21 HvButton - ghost - 14 MuiButton - flat - 23 MuiButton - flatPrimary - 24 MultiButton - btnBase - 150 MultiButton - btnSecondary - 152 MultiButton - iconWidth - ...","
```

http://localhost:9002/iframe.html?id=coremultibutton--icononlyverticalmultipleselection
``` 
   "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
   "type": "error",
   "typeCode": 1,
   "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
   "context": "<button class= MuiButtonBase - root - 44 MuiButton - root - 18 HvButton - root - 9 MuiButton - outlined - 26 MuiButton - outlinedPrimary - 27 HvButton - secondary - 12 MultiButton - btnBase - 150 MultiButton - btnSecondary - 152 MultiButton - iconWidth - 148 MultiButton - isSelected - 153\\ "...",
   "selector": "#map",
   "runner": "htmlcs",
 
   "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
   "type": "error",
   "typeCode": 1,
   "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
   "context": "<button class=MuiButtonBase - root - 44 MuiButton - root - 18 HvButton - root - 9 MuiButton - text - 20 MuiButton - textPrimary - 21 HvButton - ghost - 14 MuiButton - flat - 23 MuiButton - flatPrimary - 24 MultiButton - btnBase - 150 MultiButton - btnSecondary - 152 MultiButton - iconWidth - ...","
   selector ":"#
```  

http://localhost:9002/iframe.html?id=coremultibutton--icononlyverticalsingleselection
```   
     "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
     "type": "error",
     "typeCode": 1,
     "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
     "context": "<button class=\MuiButtonBase-root-44 MuiButton-root-18 HvButton-root-9 MuiButton-outlined-26 MuiButton-outlinedPrimary-27 HvButton-secondary-12 MultiButton-btnBase-150 MultiButton-btnSecondary-152 MultiButton-iconWidth-148 MultiButton-isSelected-153\...",
     "selector": "#map",
     "runner": "htmlcs",
   
     "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
     "type": "error",
     "typeCode": 1,
     "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
     "context": "<button class=\MuiButtonBase-root-44 MuiButton-root-18 HvButton-root-9 MuiButton-text-20 MuiButton-textPrimary-21 HvButton-ghost-14 MuiButton-flat-23 MuiButton-flatPrimary-24 MultiButton-btnBase-150 MultiButton-btnSecondary-152 MultiButton-iconWidth-...",
     "selector": "#location",
     "runner": "htmlcs",
     
     "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
     "type": "error",
     "typeCode": 1,
     "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
     "context": "<button class=\MuiButtonBase-root-44 MuiButton-root-18 HvButton-root-9 MuiButton-text-20 MuiButton-textPrimary-21 HvButton-ghost-14 MuiButton-flat-23 MuiButton-flatPrimary-24 MultiButton-btnBase-150 MultiButton-btnSecondary-152 MultiButton-iconWidth-...",
     "selector": "#map1",
     "runner": "htmlcs",
    
     "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
     "type": "error",
     "typeCode": 1,
     "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
     "context": "<button class=\MuiButtonBase-root-44 MuiButton-root-18 HvButton-root-9 MuiButton-text-20 MuiButton-textPrimary-21 HvButton-ghost-14 MuiButton-flat-23 MuiButton-flatPrimary-24 MultiButton-btnBase-150 MultiButton-btnSecondary-152 MultiButton-iconWidth-...",
     "selector": "#location1",
     "runner": "htmlcs",
    
     "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
     "type": "error",
     "typeCode": 1,
     "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
     "context": "<button class=\MuiButtonBase-root-44 MuiButton-root-18 HvButton-root-9 MuiButton-text-20 MuiButton-textPrimary-21 HvButton-ghost-14 MuiButton-flat-23 MuiButton-flatPrimary-24 MultiButton-btnBase-150 MultiButton-btnSecondary-152 MultiButton-iconWidth-...",
     "selector": "#map2",
     "runner": "htmlcs",
    
     "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
     "type": "error",
     "typeCode": 1,
     "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
     "context": "<button class=\MuiButtonBase-root-44 MuiButton-root-18 HvButton-root-9 MuiButton-text-20 MuiButton-textPrimary-21 HvButton-ghost-14 MuiButton-flat-23 MuiButton-flatPrimary-24 MultiButton-btnBase-150 MultiButton-btnSecondary-152 MultiButton-iconWidth-...",
     "selector": "#location2",
     "runner": "htmlcs",
 ```
